backend/auth-service/cp-trace/catchpoint-config-staging.yml

```diff
@@ -0,0 +1,19 @@
+catchpoint:
+  apikey: <API_KEY>
+  application:
+    name: auth-service
+    classname: java
+  report:
+    rest:
+      baseurl: https://collector-staging.tracing.catchpoint.net/v1/
+  trace:
+    instrument:
+      traceableconfig1: com.sqlcinema.backend.authservice.services.*.*[traceLineByLine=true]
```

Comment:
AFAIK, This config is not applicable for OTel SDK. If so, can be removed in config file